src/main/app/claims/responseModelConverter.ts

```diff
@@ -260,4 +264,102 @@ export class ResponseModelConverter {
     }
     return children
   }
+
+  private static convertIncomes (income: MonthlyIncome): Income[] {
+    if (!income) {
+      return undefined
+    }
+    const incomes: Income[] = []
+    if (income.salarySource && income.salarySource.populated) {
+      incomes.push({
+        type: IncomeType.JOB,
+        frequency: income.salarySource.schedule.value as PaymentFrequency,
+        amountReceived: income.salarySource.amount,
+        otherSource: undefined
+      })
+    }
+    if (income.universalCreditSource && income.universalCreditSource.populated) {
+      incomes.push({
+        type: IncomeType.UNIVERSAL_CREDIT,
+        frequency: income.universalCreditSource.schedule.value as PaymentFrequency,
+        amountReceived: income.universalCreditSource.amount,
+        otherSource: undefined
+      })
+    }
+    if (income.jobseekerAllowanceIncomeSource && income.jobseekerAllowanceIncomeSource.populated) {
+      incomes.push({
+        type: IncomeType.JOB_SEEKERS_ALLOWANCE_INCOME_BASES,
+        frequency: income.jobseekerAllowanceIncomeSource.schedule.value as PaymentFrequency,
+        amountReceived: income.jobseekerAllowanceIncomeSource.amount,
+        otherSource: undefined
```

Comment:
otherSource should be optional in the types and not required to be put here if its always undefined,